public/index.html

```diff
@@ -548,6 +554,37 @@ <h4>TCP/UDP Remote control</h4>
 										<p>Press page 1 bank 2<br>
 											<code>BANK-PRESS 1 2</code></p>
 
+										<h4>HTTP Remote control</h4>
+										<p>Remote triggering can be done by sending HTTP Requests to the same IP and port Companion is running on.</p>
+										<p><strong>Commands:</strong>
+											<ul>
+												<li><code>/press/bank/</code> &lt;page&gt;<code>/</code>&lt;bank&gt;<br>
+												<i>Press and release a button (run both down and up actions)</i></li>
+												<li><code>/style/bgcolor/</code>&lt;page&gt;<code>/</code>&lt;bank&gt;<code>?text=</code>&lt;bgcolor HEX&gt;<br>
```

Comment:
This does not look correct to me.
from a skim of the code it looks like it should be `/style/bank/<page>/<bank>?bgcolor=<hex>` 
etc

Reply:
Good catch! I committed to my own repo, does it look correct now or do I have to do anything more to make it appear here?

Reply:
It shows up here too :)